<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="binningBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>125</width>
      <height>99</height>
     </rect>
    </property>
    <property name="title">
     <string>Binning</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="binningHorizontalRadio">
       <property name="text">
        <string>Horizontal</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="binningVerticalRadio">
       <property name="text">
        <string>Vertical</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="digitalGainBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>125</width>
      <height>104</height>
     </rect>
    </property>
    <property name="title">
     <string>Digital Gain</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="digitalGainOffRadio">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="digitalGainSpin"/>
       </item>
       <item>
        <widget class="QLabel" name="digitalGainLabel">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="takePictureButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Picture</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="automaticControlBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Automatic Control (Manual) in Î¼s</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>341</width>
       <height>200</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QFrame" name="automaticControlFrame">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="automaticControlAECLabel">
               <property name="text">
                <string>AEC</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="automaticControlAECSpin">
               <property name="maximum">
                <number>5096</number>
               </property>
               <property name="value">
                <number>2000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="automaticControlAECSlider">
             <property name="maximum">
              <number>5096</number>
             </property>
             <property name="value">
              <number>2000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="automaticControlVTSLabel">
               <property name="text">
                <string>VTS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="automaticControlVTSSpin">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>5096</number>
               </property>
               <property name="value">
                <number>5096</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="automaticControlVTSSlider">
             <property name="maximum">
              <number>5096</number>
             </property>
             <property name="value">
              <number>5096</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="automaticControlAGCLabel">
               <property name="text">
                <string>AGC</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="automaticControlAGCSpin">
               <property name="maximum">
                <number>64</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="automaticControlAGCSlider">
             <property name="maximum">
              <number>64</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>More Options</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="ispButton">
         <property name="text">
          <string>ISP/ISP Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="detectPixelCancellationButton">
         <property name="text">
          <string>Detect Pixel Cancellation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="autoWhiteBalanceButton">
         <property name="text">
          <string>Auto White Balance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yuvGammaButton">
         <property name="text">
          <string>YUV Gamma</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="rawGammaButton">
       <property name="text">
        <string>Raw Gamma</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>117</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Configuration</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Main Tab</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="automaticControlTab">
     <attribute name="title">
      <string>Automatic Control</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>249</y>
      <width>116</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Configuration</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lensCorrectionRadio">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>45</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Lens Correction</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="deNoiseRadio">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>79</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>De Noise</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="evenOddRemovingRadio">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>113</y>
      <width>143</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Even Odd Removing</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="colorInterpolationRadio">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>147</y>
      <width>135</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Color Interpolation</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="blackLevelCalibrationRadio">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>181</y>
      <width>156</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Black Level Calibration</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="vapControlRegisterRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>148</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>VAP Control Register</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="previewPictureButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Picture</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="generalBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>179</width>
      <height>186</height>
     </rect>
    </property>
    <property name="title">
     <string>General</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="mirrorRadio">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mirror</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="flipRadio">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Flip</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nightModeRadio">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Night Mode</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="compressionRadio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Compression</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="specialDigitalEffectsRadio">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Special Digital Effects</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="manualModeAECAGCControlBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>128</height>
     </rect>
    </property>
    <property name="title">
     <string>AEC Manual Control</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QRadioButton" name="manualModeAECRadio">
       <property name="text">
        <string>AEC Manual</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="manualModeAGCRadio">
       <property name="text">
        <string>AGC Manual</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>lensCorrectionRadio</zorder>
   <zorder>deNoiseRadio</zorder>
   <zorder>evenOddRemovingRadio</zorder>
   <zorder>colorInterpolationRadio</zorder>
   <zorder>blackLevelCalibrationRadio</zorder>
   <zorder>vapControlRegisterRadio</zorder>
   <zorder>binningBox</zorder>
   <zorder>digitalGainBox</zorder>
   <zorder>takePictureButton</zorder>
   <zorder>automaticControlBox</zorder>
   <zorder>groupBox</zorder>
   <zorder>saveButton</zorder>
   <zorder>loadButton</zorder>
   <zorder>tabWidget</zorder>
   <zorder>previewPictureButton</zorder>
   <zorder>generalBox</zorder>
   <zorder>manualModeAECAGCControlBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera_GUI">
    <property name="title">
     <string>Camera GUI</string>
    </property>
   </widget>
   <addaction name="menuCamera_GUI"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>automaticControlAGCSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlAGCSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automaticControlAGCSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlAGCSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automaticControlAECSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlAECSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automaticControlAECSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlAECSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automaticControlVTSSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlVTSSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>automaticControlVTSSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>automaticControlVTSSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
